src/services/API/user.js

```diff
@@ -0,0 +1,59 @@
+import { client } from "@/services/API/rootClient.js";
```

Comment:
This is a new file? Why is it needed?

Reply:
It's not a new file by logic. I just renamed it. It was called `checkUser.js`. Now renamed it to `user.js` since all the user related APIs are here.